packages/presence/README.md

```diff
@@ -32,143 +19,184 @@ yarn add @solid-primitives/presence
 pnpm add @solid-primitives/presence
 ```
 
-## How to use it
+A small, reactive utility to exert stronger control over an element's presence in the DOM in order to apply enter and exit transitions/animations.
 
-### `createPresence` boolean example
+## How to use it?
 
-```tsx
-const FirstExample = () => {
-  const [showStuff, setShowStuff] = createSignal(true);
-  const { isVisible, isMounted } = createPresence(showStuff, {
-    transitionDuration: 500,
-  });
+```typescript
+const App = () => {
+  const [show, setShow] = createSignal(true);
+  const state = createPresence(show, { duration: 300, initialRun: true });
```

Comment:
Since we need it in any case, wouldn't it be better to provide `state.mounted()`, which you can use as `<Show when={state.mounted()}>`?